content/oci/oci.go

```diff
@@ -398,6 +402,49 @@ func (s *Store) writeIndexFile() error {
 	return os.WriteFile(s.indexPath, indexJSON, 0666)
 }
 
+// GC removes garbage from Store. The garbage to be cleaned are:
+//   - unreferenced (dangling) blobs in Store which have no predecessors
+//   - garbage blobs in the storage whose metadata is not stored in Store
+func (s *Store) GC(ctx context.Context) error {
```

Comment:
Since `GC` is a time-consuming operation, could you add some checkpoint for context cancelling? Also, we need to make sure that `GC` is an idempotent method.

Reply:
Added context cancelling checkpoints in the loops.